### ISSUE TITLE
Cloudfront logs: doesn't auto-detect Host properly

### ISSUE THREAD

**Issue by cjeanneret** [piwik/piwik-log-analytics#132]
Hello,

While trying to give some cloudfront logs to piwik via the import_logs script, I keep kailing with this error message:
```
Fatal error: the selected log format doesn't include the hostname: you must specify the Piwik site ID with the --idsite argument
```
After some searches, it appears this test raises it:
https://github.com/piwik/piwik-log-analytics/blob/45b09932404359f98bd44b43b09ea162bd0e3ca5/import_logs.py#L1509

A further step shows the tested content isn't really what I'm expecting:
adding a ```print format.regex.replay_tracking```, it doesn't really show "host":
```
{'status': 6, 'referrer': 7, 'x_edge_location': 2, 'x_edge_result_type': 10, 'generation_time_secs': 12, 'query_string': 9, 'length': 3, 'user_agent': 8, 'ip': 4, 'date': 1, 'path': 5, 'x_edge_request_id': 11}
```

Hence, the check just fails…

Any thought?

Cheers,

C.

**Comment by jloh** [comment]
Yeah, it looks like the Cloudfront regex assumes that you don't have a custom hostname on Cloudfront?

**Comment by mattab** [comment]
ping @jloh @cjeanneret See discussion in https://github.com/piwik/piwik-log-analytics/pull/133#issuecomment-309619097 - would be great to get this one merged :+1:
